pkg/awsclient/tags.go

```diff
@@ -42,6 +42,10 @@ func (t *AWSAccountOperatorTags) GetEC2Tags() []*ec2.Tag {
 	for _, tag := range t.Tags {
 		tags = append(tags, &ec2.Tag{Key: aws.String(tag.Key), Value: aws.String(tag.Value)})
 	}
+
+	//make sure the ec2 instance has a descriptive name to avoid customer confusion
+	tags = append(tags, &ec2.Tag{Key: aws.String("Name"), Value: aws.String("red-hat-region-init")})
```

Comment:
The following tag also needs to be added: `red-hat-managed: true`

Reply:
what does `red-hat-managed` have to do with OSD-7011? Should that be a separate card and PR, or is there some relation to this change?

Reply:
I discussed with the v1alpha1 team aligning all tags across the verifier and the aws-account-operator with OSD-7011 as the card I saw adding tags to the operator. I assumed OSD-7011 had been updated with the need for this tag, but it wasn't. As it stands now, you're right, 7011 doesn't mention this tag.

Reply:
suggest opening a new PR for `red-hat-managed: true`. I opened:
[OSD-9949] Add the `red-hat-managed: true` tag to AAO-created EC2 instances